README.rst

```diff
@@ -16,7 +16,8 @@ Warehouse
      :alt: Requirements Status
 
 Warehouse is a next generation Python Package Repository designed to replace
-the legacy code base that currently powers `PyPI <https://pypi.python.org/>`_.
+the legacy code base that currently powers `PyPI <https://pypi.python.org/>`_
+(PyPI's source code `lives separately on BitBucket <https://bitbucket.org/pypa/pypi/>`_).
```

Comment:
I think this might read better as:

```
Warehouse is a next generation Python Package Repository designed to replace the legacy code base that currently powers PyPI (whose source code lives separately on BitBucket).
```